packages/library-sync/build.gradle.kts

```diff
@@ -62,6 +62,21 @@ kotlin {
                 // Runtime holds annotations, etc. that has to be exposed to users
                 // Cinterop does not hold anything required by users
                 implementation(project(":cinterop"))
+
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${Versions.coroutines}") {
+                    version { strictly(Versions.coroutines) }
+                }
+                implementation("org.jetbrains.kotlinx:atomicfu:${Versions.atomicfu}")
+                implementation("org.jetbrains.kotlinx:kotlinx-serialization-core:${Versions.serialization}")
+                implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:${Versions.serialization}")
+
+                implementation("io.ktor:ktor-client-core:${Versions.ktor}")
+                implementation("io.ktor:ktor-client-cio:${Versions.ktor}")
```

Comment:
We are already specifying engines per target below, why pulling CIO in the common?

Reply:
The `realm-client-cio` seems to work for macos even though not listed in https://ktor.io/docs/http-client-engines.html#desktop, but from your advice I removed the common client to follow the official documentation.